File: Back-end/biblioteca/src/main/java/com/managerServices/biblioteca/dao/repository/UserRepository.java
package com.managerServices.biblioteca.dao.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import com.managerServices.biblioteca.dto.Usuario;

public interface UserRepository extends JpaRepository<Usuario, Integer>{
	
	public Usuario findByEmailAndSenha(String mail,String senha);
	
	public Usuario findByIdAndSenha(int id, String senha);
}

File: Back-end/biblioteca_kotlin/library/src/main/kotlin/com/biblioteca/library/repository/UserRepository.kt
package com.biblioteca.library.repository

import org.springframework.data.jpa.repository.JpaRepository
import com.biblioteca.library.dto.Usuario

interface UserRepository : JpaRepository<Usuario, Int> {
	
	fun findByEmailAndSenha(mail: String?, senha: String?) : Usuario
	
	fun findByIdAndSenha(id: Int, senha: String): Usuario
}